soc/intel/intel_adsp/ace/ace-link.ld

```diff
@@ -171,6 +171,16 @@ SECTIONS {
    */
   .imrdata : ALIGN(4096) {
     *(.imrdata .imrdata.*)
+  } >imr
+
+  /* Cold code in IMR memory */
+  .cold : ALIGN(4096) {
+    *(.cold .cold.* )
```

Comment:
Seems like this is basically identical to the pre-existing .imrdata and should be unified, no?